src/parse.jl

```diff
@@ -172,7 +172,7 @@ function parseURLForDispatch(url, region, nowarn)
     ok, accelerate, host, bucket, reg, key = parseAWSBucketRegionKey(url)
     region = isempty(reg) ? region : reg
     if ok && region === nothing
-        nowarn || @warn "`region` keyword argument not provided to `CloudStore.get` and undetected from url.  Defaulting to `us-east-1`"
+       # nowarn || @warn "`region` keyword argument not provided to `CloudStore.get` and undetected from url.  Defaulting to `us-east-1`"
```

Comment:
Maybe keep as `@debug`?

Reply:
This isn't meant to stay in; it's just super annoying during benchmarking

Reply:
Doh, I should just pass `nowarn` in my benchmark script!